src/metabase/query_processor/middleware/permissions.clj

```diff
@@ -89,16 +90,29 @@
   (doseq [{query :dataset_query} (qp.u.tag-referenced-cards/tags-referenced-cards outer-query)]
     (check-query-permissions* query)))
 
-(s/defn ^:private check-query-permissions*
+(def ^:dynamic *user-can-read-dashboard*
+  "Used to allow users looking at a dashboard to view chained filters."
```

Comment:
This isn't actually specific to chain filters right? i.e. all field value lookups go through those endpoints and thus have this binding set?